lib/nimbusRecipeCollection.ts

```diff
@@ -21,7 +21,10 @@ async function updateBranchesCTR(recipe: NimbusRecipe): Promise<BranchInfo[]> {
         // Looker being case sensitive
         const ctrPercent = await getCTRPercent(
           branchInfo.id.toUpperCase(),
-          branchInfo.template
+          branchInfo.template!,
+          undefined,
```

Comment:
Agreed, I'll leave a comment. Also, I left the channel undefined because that's how it is when the dashboard link is being defined for branches (see [here](https://github.com/mozilla/skylight/blob/main/lib/nimbusRecipe.ts#L185-L186)).